editor/src/components/canvas/controls/select-mode/grid-gap-control-helpers.tsx

```diff
@@ -0,0 +1,35 @@
+import type { CanvasVector } from '../../../../core/shared/math-utils'
+import { windowPoint } from '../../../../core/shared/math-utils'
+import { Modifier } from '../../../../utils/modifiers'
+import type { EditorDispatch } from '../../../editor/action-types'
+import CanvasActions from '../../canvas-actions'
+import { createInteractionViaMouse, gridGapHandle } from '../../canvas-strategies/interaction-state'
+import { windowToCanvasCoordinates } from '../../dom-lookup'
+import type { Axis } from '../../gap-utils'
+
+export function startInteraction(
```

Comment:
I think it'd be better to give this a more specific name, e.g. `startGapControlInteraction` (or whatever makes most sense to you) since it's exported

Reply:
oh yes this is a very good shout